app/src/main/res/values-pt-rBR/strings.xml

```diff
@@ -200,29 +202,31 @@
   <string name="size_gb">%s GB</string>
   <string name="correct">Correto!</string>
   <string name="topic_prefix">Tópico: %s</string>
-  <plurals name="chapter_count" fuzzy="true">
-    <item quantity="one">%s Capítulo</item>
+  <string name="selected_image_region_selection_content_description">Mostrando imagem %1$s</string>
+  <string name="unselected_image_region_selection_content_description">Selecione %1$s imagem</string>
+  <plurals name="chapter_count">
+    <item quantity="one">1 Capítulo</item>
     <item quantity="other">%s Capítulos</item>
   </plurals>
-  <plurals name="story_count" fuzzy="true">
-    <item quantity="one">%s História</item>
-    <item quantity="other">%s Histórias</item>
+  <plurals name="story_count">
+    <item quantity="one">1 Story</item>
+    <item quantity="other">%s Stories</item>
   </plurals>
-  <plurals name="story_total_chapters" fuzzy="true">
-    <item quantity="one">%s de %s Capítulo Concluído</item>
-    <item quantity="other">%s de %s Capítulos Concluídos</item>
+  <plurals name="story_total_chapters">
+    <item quantity="one">%1$s of %2$s Chapter Completed</item>
+    <item quantity="other">%1$s of %2$s Chapters Completed</item>
   </plurals>
-  <plurals name="lesson_count" fuzzy="true">
-    <item quantity="one">%s Lição</item>
-    <item quantity="other">%s Lições</item>
+  <plurals name="lesson_count">
+    <item quantity="one">1 Lesson</item>
+    <item quantity="other">%s Lessons</item>
   </plurals>
-  <plurals name="completed_story_count" fuzzy="true">
-    <item quantity="one">%s história concluída</item>
-    <item quantity="other">%s Histórias concluídas</item>
+  <plurals name="completed_story_count">
+    <item quantity="one">1 Story Completed</item>
+    <item quantity="other">%s Stories Completed</item>
   </plurals>
-  <plurals name="ongoing_topic_count" fuzzy="true">
-    <item quantity="one">%s Tópico em andamento</item>
-    <item quantity="other">%s Tópicos em andamento</item>
+  <plurals name="ongoing_topic_count">
+    <item quantity="one">1 Topic in Progress</item>
+    <item quantity="other">%s Topics in Progress</item>
```

Comment:
While the string format/template has been corrected, somehow the latest translation has reverted to English so not sure whether to merge this.

Reply:
Same to the `profile_last_visited` time strings below.